.github/workflows/checks.yaml

```diff
@@ -0,0 +1,46 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ dev ]
+  pull_request:
+    branches: [ dev ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.allow-error }}
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: ["3.6", "3.7"]
```

Comment:
I think you should add 3.8, 3.9 and 3.10 to the list, and remove 3.6 (so support in `numpy` anymore).

Reply:
The PR already has 3.8. I tried to tried to respect the allow the "allow-failures" for 3.8 as specified in travis.yaml. However, I think we should not allow continue-on error and put all versions inside the matrix. What do you think?

Reply:
I agree. Indeed v3.8 is failing due to an old pytest version. In the PR https://github.com/elixir-cloud-aai/TEStribute/pull/86 pytest is being upgraded to fix the problem

Reply:
Perfect. I'll fix it and do a new commit.